stock_orderpoint_generator/models/orderpoint_template.py

```diff
@@ -82,6 +82,20 @@ class OrderpointTemplate(models.Model):
     qty_forecast = fields.Float(compute=False, store=False)
     product_category_id = fields.Many2one(related=False, store=False)
 
+    allowed_warehouse_ids = fields.One2many(
```

Comment:
@astirpe Could you put those changes that are enhancement in a separate commit in order to be able to backport them (and separate from real migration) ? Thanks

Reply:
@rousseldenis They are part of the real migration, not enhancement. Odoo would raise an error without this part. I think this should be backported as well.

Reply:
That's why a separate commit is more convenient

Reply:
Not sure I get your point. When backporting from V17 to V16, other parts of the code should be changed, like the qty_done and the views, but not this part that could be kept as it is.